examples/apollo-federation/accounts/index.ts

```diff
@@ -0,0 +1,87 @@
+import {
```

Comment:
I think we should remove the federations example from this PR. We can use it later in separate PR when adding federation examples when the extensions stuff will be resolved. For now it should be usable with directives and addResolversToSchema workaround.